Gems/Atom/RHI/Vulkan/Code/Source/RHI/Queue.cpp

```diff
@@ -74,6 +78,34 @@ namespace AZ
                 submitInfo = {};
                 submitInfo.sType = VK_STRUCTURE_TYPE_SUBMIT_INFO;
                 submitInfo.pNext = nullptr;
+
+                if ((fenceToSignal && fenceToSignal->GetFenceType() == FenceType::TimelineSemaphore) || !fencesToWaitFor.empty())
+                {
+                    timelineSemaphoresSubmitInfos.sType = VK_STRUCTURE_TYPE_TIMELINE_SEMAPHORE_SUBMIT_INFO;
+                    timelineSemaphoresSubmitInfos.pNext = nullptr;
+                    submitInfo.pNext = &timelineSemaphoresSubmitInfos;
+
+                    if ((fenceToSignal && fenceToSignal->GetFenceType() == FenceType::TimelineSemaphore))
+                    {
+                        uint64_t value = fenceToSignal->GetPendingValue();
+                        timelineSemaphoresSubmitInfos.signalSemaphoreValueCount = 1;
+                        timelineSemaphoresSubmitInfos.pSignalSemaphoreValues = &value;
```

Comment:
Am I seeing this wrong and this is a pointer to a local variable?

Reply:
Oh yes, no idea how this worked at all. Fixed.